### ISSUE TITLE
Used local branch when execute `new` action of `GinBranch` to remote branch

### ISSUE THREAD

**Issue by kuuote** [lambdalisue/gin.vim#96]
# Problem

Used local branch when execute `new` action of `GinBranch` to remote branch

# Enviroment

- NVIM v0.10.0-dev-706+gcdb758ef4-dirty
- git version 2.41.0
- https://github.com/vim-denops/denops.vim/commit/22e7ae3d712f190b3f46483e6b9dd50a99a9abbd
- https://github.com/lambdalisue/gin.vim/commit/fd28fa823bb41143d9af822b27f82bc873a85021

# To Reproduce

1. introduce `init.vim`.

```vim
set rtp+=/path/to/vim-denops/denops.vim
set rtp+=/path/to/lambdalisue/gin.vim

cd <sfile>:p:h
let s:cmd =<< END
  rm -rf foo 
  rm -rf bar
  git init foo
  cd foo
  git switch --orphan foo
  git commit --allow-empty -m hoge
  git switch -c bar
  cd -
  git clone foo bar
  cd bar
  git branch foo
END
call system('sh', s:cmd)
cd bar

autocmd User DenopsPluginPost:gin ++nested GinBranch -a
```

2. launch Neovim `nvim -u init.vim`.
3. Execute new action at `remotes/origin/foo` line.
4. Create a new branch from local branch `foo`.

# Expected
Create a new branch from remote branch `remotes/origin/foo`.

**Comment by lambdalisue** [comment]
@kuuote Could you try https://github.com/lambdalisue/gin.vim/pull/97?

**Comment by kuuote** [comment]
@lambdalisue Its fixed at #97. Thanks.
